spec/sections/standard.adoc

```diff
@@ -348,12 +348,13 @@ Critical implementation details related to `multiple` member should be reference
 |`name`
 |Required
 2+|MUST be a string of only alphabetic, dash or underscore characters (defined by the regex† `^[a-zA-Z_-]+$`) indicating
-the environment variable name that will be injected by the processing platform for job consumption. Refer to
+the environment variable name that will be injected by the executor† for job consumption. Refer to
 <<variable-injection>> for details on environment variable available at execution time.
 
-|`mediaType`
-|Required
-2+|MUST be an array of strings that MUST indicate the IANA Media types that the job accepts.
+|`mediaTypes`
+|Optional
+2+|MUST be an array of strings that MUST indicate the IANA Media types that the job accepts. If omitted, executor† will
+not be able to perform validation.
```

Comment:
Suggest changing `If omitted, executor† will not be able to perform validation.` to `An executor† MAY use any provided media types to report validation warnings to the user in the case of mismatched types.`

Reply:
+